flink-test-utils-parent/flink-test-utils-junit/src/main/java/org/apache/flink/testutils/s3/S3TestCredentials.java

```diff
@@ -44,7 +44,8 @@
 	 * of this JVM.
 	 */
 	public static boolean credentialsAvailable() {
-		return S3_TEST_BUCKET != null && S3_TEST_ACCESS_KEY != null && S3_TEST_SECRET_KEY != null;
```

Comment:
Add a simple utility function `boolean isNotEmpty(@Nullable String)` that you call 3 times. This is both more readable and would've prevented bugs like in the second line below where `S3_TEST_SECRET_KEY.isEmpty()` is missing a negation.

Reply:
Thanks a lot. That's a very good idea :)